Jenkinsfile.cf

```diff
@@ -53,28 +53,38 @@ pipeline {
         }
         stage('Publish') {
             steps {
-                // sh 'export TAG=$(git log --pretty=format:'%h' -n 1)'
                 sh 'docker login -u charlesfc001 -p ${DOCKER_PASS}'
                 sh 'docker tag compiler charlesfc001/compiler:${GIT_COMMIT_HASH}'
                 sh 'docker push charlesfc001/compiler:${GIT_COMMIT_HASH}'
             }
         }
         stage('DeployToDev') {
             steps {
-                // SH 'echo TAG_VERSION=${GIT_COMMIT_HASH} > .env.version'                
-                sh 'echo ${TAG_VERSION}'                
+                sh 'echo ${TAG_VERSION}'
                 sh 'export TAG_VERSION=${GIT_COMMIT_HASH} && docker-compose -f docker-compose.cj.yaml up -d'
             }
         }
         stage('DeployToAUTO') {
             steps {
-                sh 'DOCKER_HOST=ssh://$TARGET_HOST export TAG_VERSION=${GIT_COMMIT_HASH} && docker-compose -f docker-compose.cj.yaml up -d'
+                sh 'export TAG_VERSION=${GIT_COMMIT_HASH} && DOCKER_HOST=ssh://$TARGET_HOST docker-compose -f docker-compose.cj.yaml up -d'
             }
         }
-    } 
+    }
     post {
         always {
             sh 'docker rmi $(docker images -f "dangling=true" -q)'
         }
+        success {
```

Comment:
Use environment variables to improve your email notification. always {}
Pipeline for ${BUILD_NUMBER} - has been executed - Status: ${BUILD_STATUS}
The Convert service build was executed ${BUILD_STATUS}
This is the log link